### ISSUE TITLE
Print out entire ssh command issued in debug mode?

### ISSUE THREAD

**Issue by efrecon** [docker/machine#1207]
Version 0.2 of docker prints out the entire `ssh` command that is being issued as part of the debug output. This behaviour seems to have been removed from the coming 0.3. Can it be put back?

Being able to access the `ssh` command is handy when automating machine, which I do from [machinery](https://github.com/efrecon/machinery). For example, machinery can use `rsync` to copy files between the host and the machine that is being created. To copy, `rsync` needs to know the exact `ssh` command to execute and picking up the command that is used by machine itself guarantees that it will continue to work in future versions.

**Comment by efrecon** [comment]
Now that I've read #909, I understand why the output has gone... So let me rephrase the request instead. Could `inspect` return all the information necessary to connect to the machine using an external `ssh` client? At present, for example, the full path to the identity file to pass (for example through the `-i` option of `ssh`) is not contained in the JSON description. Instead, what can be found there is the directory containing the file.

**Comment by ehazlett** [comment]
I think we could add this to inspect.  

/cc @nathanleclaire  @hairyhenderson thoughts?

**Comment by nathanleclaire** [comment]
Hm, it won't cover every single use case, but we just switched back to shelling out to ssh by default (native Go impl as a backup), and you should be able to see the `ssh` invocations using `-D` if you want.  I usually just copy-paste them if I want to run them manually.  Would that cover your use case?

**Comment by nathanleclaire** [comment]
I do, longer-term, want to add a `--delta` / `--rsync` flag to `docker-machine` scp which will construct and run a reasonable rsync command for you, but it will take a minute.

**Comment by hairyhenderson** [comment]
Ok, yeah. I think it's a good idea to get the SSH identity's path into the `inspect` output.

@ehazlett, I'll get started on this ;)

**Comment by efrecon** [comment]
Running `rsync` requires that you have installed in the virtual machine. This is one of the things that I had to fix for getting the `rsync` facility working in [machinery](https://github.com/efrecon/machinery). Fortunately, in [boot2docker](https://github.com/boot2docker/boot2docker), it is only a matter of `tce-load -wi rsync`, but you will have to run similar commands for all the VM OSes that you want to support.

**Comment by nathanleclaire** [comment]
This should be supported now - I can see all SSH commands used when I run a command.
